src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs

```diff
@@ -57,8 +68,13 @@ private CancellationToken StartIISExpress(Uri uri, string contentRoot)
                 // Pass on the applicationhost.config to iis express. With this don't need to pass in the /path /port switches as they are in the applicationHost.config
                 // We take a copy of the original specified applicationHost.Config to prevent modifying the one in the repo.
 
+                // The nupkg build of ANCM does not support Win7. https://github.com/aspnet/AspNetCoreModule/issues/40.
+                // If Win7 were supported we could just hardcode "aspnetcore.dll" in the config files.
+                var ancmPath = IsWin8orLater ? "aspnetcore.dll" : @"%ProgramFiles%\IIS Express\aspnetcore.dll";
```

Comment:
Also, the old path looks like it's `%SystemRoot%\System32\inetsrv\aspnetcore.dll` why are we changing that now?